third_party/move/move-compiler-v2/src/options.rs

```diff
@@ -25,29 +31,36 @@ pub struct Options {
         num_args = 0..
     )]
     pub dependencies: Vec<String>,
+
```

Comment:
Has this been done by the auto formatter? Please note that there is no agreement in general what is the better style, it actually pretty much depends on the IDE and font you are using. Its not necessarily better now. I would leave such decisions of formatting at the author and not enforce my private style tastes.

Reply:
I just can't read it the other way.  Call it an accessibility issue for the partially-blind.